sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/RequestResponseModelTests.cs

```diff
@@ -74,7 +102,7 @@ public async Task TokenIssuanceStartObjectModelNullResponseTest()
                 {
                     request.Response = null;
 
-                    eventsResponseHandler.SetValueAsync(request.Completed().Result, CancellationToken.None);
+                    eventsResponseHandler.SetValueAsync(request.Completed(), CancellationToken.None);
```

Comment:
why us this async?

Reply:
This object implements the interface IValueBinder which requires this implementation.

Reply:
what does SetValueAsync return? If it is a task, are we intentionally not awaiting it?

Reply:
Updated to await